src/main/java/com/afornalik/ox/Player.java

```diff
@@ -1,12 +1,10 @@
-package com.afornalik.ox.model.player;
+package com.afornalik.ox;
 
-import com.afornalik.ox.model.board.FieldStatus;
-
-public class Player {
+class Player {
 
     private final String name;
     private int score;
-    private final FieldStatus sign;
+    private final Field sign;
     private final boolean first;
```

Comment:
Czyli mamy dwa rodzaje graczy. Pierwszego i NiePierwszego. To może tak interfejs Gracz i dwie implementacje? Czym się te warianty od siebie różnią? Bo jeśli tym, że jednego możemy zapytać "jesteś pierwszy" i odpowiada tak a drugi odpowiada nie... to słabo trochę.

Reply:
Zmieniłem typ z boolean na int tak, aby odzwierciedlało kolejność graczy.